manifests/copr.pp

```diff
@@ -0,0 +1,79 @@
+# Define: yum::copr
+#
+# This definition manages Copr (Cool Other Package Repo) repositories.
+#
+# Parameters:
+#   [*repo*]      - name of repository, defaults to title
+#   [*ensure*]    - specifies if repo should be enabled,
+#                   disabled or removed
+#
+# Actions:
+#
+# Requires:
+#   RPM based system
+#
+# Sample usage:
+#   yum::copr { 'copart/restic':
+#     ensure  => 'enabled',
+#   }
+#
+define yum::copr (
+  String                                 $repo   = $title,
```

Comment:
please add puppet-strings like documentation for the new parameters

Reply:
Done in the last force-push, really appreciate all the feedback from review, since I'm not yet used to most of the scaffolding and test infrastructure, and you are pushing me to read into the correct things :+1: .